functions/join.go

```diff
@@ -330,7 +330,10 @@ func JoinNetwork(flags *viper.Viper) (*config.Node, *config.Server, error) {
 	}
 	fmt.Println("checking for version compatibility ", joinResponse.ServerConfig.Version)
 	if !IsVersionComptatible(joinResponse.ServerConfig.Version) {
-		return nil, nil, errors.New("incompatible server version")
+		logger.Log(1, "server/client version mismatch, trying to update to server version: ", joinResponse.ServerConfig.Version)
```

Comment:
IsVersionComtatible fails if the server version is less that v0.18.0 and client version is higher....  In this case we do not want to downgrade netclient